src/modules/75-cf/pages/feature-flags-detail/flag-pipeline-tab/components/available-pipelines-drawer/AvailablePipelinesDrawer.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+import { Drawer, Position } from '@blueprintjs/core'
+import {
+  Color,
+  Container,
+  FontVariation,
+  Layout,
+  Text,
+  ExpandingSearchInput,
+  Button,
+  ButtonVariation
+} from '@harness/uicore'
+import React, { useCallback, useMemo, useState } from 'react'
+import { useParams } from 'react-router-dom'
+import { useStrings } from 'framework/strings'
+import { FeatureAvailablePipeline, useGetAvailableFeaturePipelines } from 'services/cf'
+import useActiveEnvironment from '@cf/hooks/useActiveEnvironment'
+import { ContainerSpinner } from '@common/components/ContainerSpinner/ContainerSpinner'
+import PipelineCard from '../pipeline-card/PipelineCard'
+import css from './AvailablePipelinesDrawer.module.scss'
+interface AvailablePipelinesDrawerProps {
+  isOpen: boolean
+  flagIdentifier: string
+  onClose: () => void
+}
+
+const AvailablePipelinesDrawer: React.FC<AvailablePipelinesDrawerProps> = ({ flagIdentifier, isOpen, onClose }) => {
+  const { getString } = useStrings()
+  const { activeEnvironment: environmentIdentifier } = useActiveEnvironment()
+
+  const PAGE_SIZE = 100
+  const [selectedPipeline, setSelectedPipeline] = useState<FeatureAvailablePipeline>()
+
+  const { orgIdentifier, accountId: accountIdentifier, projectIdentifier } = useParams<Record<string, string>>()
+
+  const queryParams = useMemo(
+    () => ({
+      identifer: flagIdentifier,
+      environmentIdentifier,
+      projectIdentifier,
+      accountIdentifier,
+      orgIdentifier,
+      pageSize: PAGE_SIZE
+    }),
+    [accountIdentifier, environmentIdentifier, orgIdentifier, projectIdentifier, flagIdentifier]
+  )
+
+  const { data, loading, refetch } = useGetAvailableFeaturePipelines({
+    queryParams,
+    debounce: 500
+  })
+
+  const onSearchInputChanged = useCallback(
+    value => {
+      setSelectedPipeline(undefined)
+      refetch({ queryParams: { ...queryParams, pipelineName: value, pageNumber: 0 }, debounce: 500 })
+    },
+    [queryParams, refetch]
+  )
+
+  return (
+    <Drawer position={Position.RIGHT} isOpen={isOpen} onClose={onClose} className={css.drawer}>
+      <Layout.Vertical padding="xxlarge" spacing="medium">
+        <Container flex={{ justifyContent: 'space-between', alignItems: 'center' }}>
+          <Text font={{ variation: FontVariation.H3 }}>{getString('cf.featureFlags.flagPipeline.drawerTitle')}</Text>
+
+          <Button minimal icon="cross" withoutBoxShadow onClick={onClose} data-testid="close-drawer-button" />
```

Comment:
Best to add the `icon` variation and a `aria-label` to explain what this does.
```suggestion
          <Button
            icon="cross"
            variation={ButtonVariation.ICON}
            aria-label={getString('close')}
            onClick={onClose}
            data-testid="close-drawer-button"
          />
```

Another benefit of the `aria-label` is that it works with `getByRole` as the accessible name of the button.

Reply:
fixed - weird the linter doesn't call this out